cmd/export-client/main.go

```diff
@@ -54,18 +52,18 @@ func main() {
 	client.LoggingClient.Info("Service dependencies resolved...")
 	client.LoggingClient.Info(fmt.Sprintf("Starting %s %s ", internal.ExportClientServiceKey, edgex.Version))
 
-	http.TimeoutHandler(nil, time.Millisecond*time.Duration(client.Configuration.Timeout), "Request timed out")
-	client.LoggingClient.Info(client.Configuration.AppOpenMsg, "")
+	http.TimeoutHandler(nil, time.Millisecond*time.Duration(client.Configuration.Service.Timeout), "Request timed out")
+	client.LoggingClient.Info(client.Configuration.Service.StartupMsg, "")
```

Comment:
I know @drasko hates magic strings ;-) Secondarily it would be easier to change if a customer wanted some type of branding message to be output -- like "Starting ACME Inc. Export Service". They wouldn't have to recompile the code then.

Reply:
There are so many things a customer will want to change, and we can't add all them. Keeping config files as clean and small as possible is more important, at least to me, than having options to change just some strings.